homeassistant/components/conversation/default_agent.py

```diff
@@ -145,6 +145,7 @@ def async_hass_started(hass: core.HomeAssistant) -> None:
     ha_start.async_at_started(hass, async_hass_started)
 
 
+# pylint: disable-next=hass-enforce-class-module
```

Comment:
This should also be allowed, since this module is a submodule of the package of the entity integration that owns the entity.

Reply:
Indeed...
It is allowed - I just forgot to remove the disable comment